docs/topics/cache.txt

```diff
@@ -875,6 +875,8 @@ of key-value pairs::
 
 Like ``cache.set()``, ``set_many()`` takes an optional ``timeout`` parameter.
 
+if insertion fails for some keys, invalid keys will be returned by ``set_many``.
```

Comment:
Could be useful to mention what you wrote about the keys being transformed in the return value. Also needs a `.. versionchanged:: 1.11` annotation as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.

I guess it's small enough not to merit a mention in the release notes.